waku/v2/peermanager/peer_manager.go

```diff
@@ -24,47 +24,49 @@ const WakuRelayIDv200 = protocol.ID("/vac/waku/relay/2.0.0")
 // PeerManager applies various controls and manage connections towards peers.
 type PeerManager struct {
 	peerConnector       *PeerConnectionStrategy
-	maxConnections      int
 	maxRelayPeers       int
 	logger              *zap.Logger
 	InRelayPeersTarget  int
 	OutRelayPeersTarget int
 	host                host.Host
-	serviceSlots        map[protocol.ID][]peer.ID
+	serviceSlots        *ServiceSlot
 	ctx                 context.Context
 }
 
-const maxRelayPeersShare = 5
-
-// const defaultMaxOutRelayPeersTarget = 10
-const outRelayPeersShare = 3
 const peerConnectivityLoopSecs = 15
-const minOutRelayConns = 10
+
+// 80% relay peers 20% service peers
+func relayAndServicePeers(maxConnections int) (int, int) {
```

Comment:
Should rename to ServiceAndRelayPeers as first return is for servicePeers. 
Also add comments or have named returns to make code readable.